cmd/nerdctl/images.go

```diff
@@ -314,8 +314,8 @@ func (key snapshotKey) add(ctx context.Context, s snapshots.Snapshotter, usage *
 
 // unpackedImageSize is the size of the unpacked snapshots.
 // Does not contain the size of the blobs in the content store. (Corresponds to Docker).
```

Comment:
Maybe we should show both the blob size and the unpacked size to avoid confusion for multi-platform images

Reply:
fixed